modules/log6x-cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -0,0 +1,170 @@
+----
+
+.Example syslog message output with `enrichment: KubernetesMinimal`
+
+[source, text]
+----
+2025-03-03T11:42:31+00:00 anliazm3-m8hrk-worker-westus-djq2j syslogaosqersyslogserverd846bb9b: namespace_name=syslog-aosqe, container_name=rsyslog, pod_name=rsyslogserver-d846bb9bc-l2tjr, message={"@timestamp":"2025-03-03T11:42:31.862624122Z","facility":"local0","hostname":"anliazm3-m8hrk-worker-westus-djq2j","kubernetes":{"annotations":{"k8s.ovn.org/pod-networks":"{\"default\":{\"ip_addresses\":[\"10.129.2.40/23\"],\"mac_address\":\"0a:58:0a:81:02:28\",\"gateway_ips\":[\"10.129.2.1\"],\"routes\":[{\"dest\":\"10.128.0.0/14\",\"nextHop\":\"10.129.2.1\"},{\"dest\":\"172.30.0.0/16\",\"nextHop\":\"10.129.2.1\"},{\"dest\":\"169.254.169.5/32\",\"nextHop\":\"10.129.2.1\"},{\"dest\":\"100.64.0.0/16\",\"nextHop\":\"10.129.2.1\"}],\"ip_address\":\"10.129.2.40/23\",\"gateway_ip\":\"10.129.2.1\"}}","k8s.v1.cni.cncf.io/network-status":"[{\n    \"name\": \"ovn-kubernetes\",\n    \"interface\": \"eth0\",\n    \"ips\": [\n        \"10.129.2.40\"\n    ],\n    \"mac\": \"0a:58:0a:81:02:28\",\n    \"default\": true,\n    \"dns\": {}\n}]","openshift.io/generated-by":"OpenShiftNewApp","openshift.io/scc":"privileged"},"container_id":"cri-o://699290ce5f3af668ebebd1789d5df9bfb1af57ff2bbbeb6241f56af845bbec87","container_image":"quay.io/openshifttest/rsyslogd-container@sha256:e806eb41f05d7cc6eec96bf09c7bcb692f97562d4a983cb019289bd048d9aee2","container_image_id":"quay.io/openshifttest/rsyslogd-container@sha256:821f81c0deea63c4f06481e3773eac3f55892c69c91b3751b0ba61fb6decb905","container_iostream":"stderr","container_name":"rsyslog","labels":{"appname":"rsyslogserver","component":"rsyslogserver","pod-template-hash":"d846bb9bc","provider":"aosqe"},"namespace_id":"ec872692-4bde-45d3-ae4b-89ba35628a3d","namespace_labels":{"kubernetes_io_metadata_name":"syslog-aosqe","pod-security_kubernetes_io_audit":"privileged","pod-security_kubernetes_io_audit-version":"v1.24","pod-security_kubernetes_io_warn":"privileged","pod-security_kubernetes_io_warn-version":"v1.24"},"namespace_name":"syslog-aosqe","pod_id":"54a774f8-9e87-4867-b285-ef868cca6032","pod_ip":"10.129.2.40","pod_name":"rsyslogserver-d846bb9bc-l2tjr","pod_owner":"ReplicaSet/rsyslogserver-d846bb9bc"},"level":"default","log_source":"container","log_type":"application","message":"rsyslogd: netstream session 0x7f1a5ad94620 from 10.129.0.92 will be closed due to error [v8.2102.0-3.fc34 try https://www.rsyslog.com/e/2078 ]","openshift":{"cluster_id":"4fd3bf05-1e95-46cf-8480-62fc9435079c","sequence":1741002167861044318},"proc_id":"-","severity":"informational","tag":"syslogaosqersyslogserverd846bb9b","timestamp":"2025-03-03T11:42:31.862624122Z"}
+----
+
+[id="cluster-logging-collector-log-forward-examples-syslog-parms_{context}"]
+== Syslog parameters
```

Comment:
We had introduce these Syslog parameters  above. I think we can drop the paragraph below

Reply:
done